vispy/visuals/isocurve.py

```diff
@@ -96,8 +106,8 @@ def set_data(self, data):
         if _HAS_MPL:
             try:
                 if not (self._X.T.shape == data.shape):
-                    raise
-            except:
+                    raise Exception
+            except Exception:
```

Comment:
One might think it could be written as:
```
if not (self._X.T.shape == data.shape):
    self._X, self.Y = np.meshgrid(...)
```
but the bare exception also would catch `AttributeError` on `self._X`, or `<something>.shape`, etc., so is that intended? I don't know, but that's why I don't like bare exception handling...